FogCreek/Glitch-Community
# Cheerful flamingo: use "Thanks" component everywhere

## Links
* Remix link: https://cheerful-flamingo.glitch.me/

## GIF/Screenshots:
![Screen Shot 2019-04-10 at 3 46 30 PM](https://user-images.githubusercontent.com/938722/55908661-07806880-5ba8-11e9-8856-a4f1d453c965.png)
![Screen Shot 2019-04-10 at 3 46 45 PM](https://user-images.githubusercontent.com/938722/55908662-07806880-5ba8-11e9-8a8f-49e31c1e5bc2.png)
![Screen Shot 2019-04-10 at 3 50 20 PM](https://user-images.githubusercontent.com/938722/55908828-6940d280-5ba8-11e9-8a8e-f22236566d31.png)


## Changes:
* use Thanks component everywhere
* get rid of redundant thanks count checking
* delete old Thanks presenter

## How To Test:
* example team: https://cheerful-flamingo.glitch.me/@glitch
* click on a team member to see the user info pop
* example user: https://cheerful-flamingo.glitch.me/@anildash
* example user with zero thankses https://cheerful-flamingo.glitch.me/user/123

## Feedback I'm looking for:
* do these look right?
* should the Emoji component have configurable sizes? (It seems like its overridden in a lot of places)
